python_src/src/lamp_py/migrations/versions/performance_manager/013_97f5fb821b8a_start_date_to_service_date.py

```diff
@@ -0,0 +1,261 @@
+"""start_date to service_date
+
+Revision ID: 97f5fb821b8a
+Revises: ced3caaeab23
+Create Date: 2023-05-15 14:54:48.190847
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "97f5fb821b8a"
+down_revision = "ced3caaeab23"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.alter_column(
+        "vehicle_events", "start_date", new_column_name="service_date"
+    )
+    op.alter_column(
+        "vehicle_trips", "start_date", new_column_name="service_date"
+    )
+
+    update_view = """
+        CREATE OR REPLACE VIEW static_service_id_lookup AS 
+        WITH mod_feed_dates AS (
+            SELECT 
+                feed_start_date::text::date AS start_date,
+                feed_end_date::text::date AS end_date,
+                "timestamp"
+            FROM public.static_feed_info
+        ), sd_match AS (
```

Comment:
Both of these views were updated to remove an erroneous CTE statement from a previous version that referenced `start_date` 

`        ), trips_start_dates AS (
            SELECT
                start_date,
                min(fk_static_timestamp) AS fk_timestamp
            FROM
                public.vehicle_trips
            GROUP BY
                start_date
        )
`